### PR TITLE
GHA fix

### PR DESCRIPTION
**Pull request by Xiangs18**

### CODE REVIEW

**Comment by Xiangs18** [comment]
In https://github.com/kbaseapps/GenomeFileUtil/pull/204/commits/f9857e806482d13b6f6b1274b81f8634a8312bc6, when tests pass, GHA passes
In https://github.com/kbaseapps/GenomeFileUtil/pull/204/commits/9d805feeac818c22c3dac1137a434b268bcd6aa9, when tests fail, GHA fails

I will squash and merge this PR as most of the commits are experiments/tests.

File: Makefile [hunk 1]
```diff
@@ -51,7 +51,6 @@ build-test-script:
 	echo 'export PYTHONPATH=$$script_dir/../$(LIB_DIR):$$PATH:$$PYTHONPATH' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'cd $$script_dir/../$(TEST_DIR)' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
 	echo 'python -m nose --with-coverage --cover-package=$(SERVICE_CAPS) --cover-html --cover-html-dir=/kb/module/work/test_coverage --cover-xml --cover-xml-file=/kb/module/work/test_coverage/coverage.xml --nocapture  --nologcapture .' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
-	echo 'cp /kb/module/.coveragerc .' >> $(TEST_DIR)/$(TEST_SCRIPT_NAME)
```
Comment (MrCreosote) [inline Makefile:54]: Why are this and the other line dealing with `.coveragerc` not needed?

Comment (Xiangs18) [reply Makefile:54]: 1. this file does not exist: https://github.com/kbaseapps/GenomeFileUtil/actions/runs/7404244710/job/20145462822#step:6:1100
2. Upload coverage to Codecov step works fine without it

Comment (MrCreosote) [reply Makefile:54]: Interesting, it looks like when coverage was added to this repo no one added a coveragerc but did add the line: https://github.com/kbaseapps/GenomeFileUtil/commit/80056995d163a1a9b551dec79841ac6cae2ef6a8

**Review by MrCreosote — APPROVED** [review]

**Review by Tianhao-Gu — APPROVED** [review]
👍

**Pull request merged**